Sources/ProtoGen/connector_service.grpc.swift

```diff
@@ -144,7 +144,7 @@ public struct Google_Firebase_Dataconnect_V1beta_ConnectorServiceNIOClient: Goog
   }
```

Comment:
Don't modify this file - its a generated file by the proto compiler.

Reply:
Fixed